src/main/java/ch/njol/skript/expressions/ExprStringCase.java

```diff
@@ -41,76 +44,118 @@
 /**
  * @author bensku
  */
```

Comment:
You can remove this because you changed a lot of things and Skript doesn't use them anymore